Building DAG of jobs...
Using shell: /bin/bash
Provided cores: 10
Rules claiming more threads will be scaled down.
Job counts:
	count	jobs
	2	STAR_align
	1	all
	2	fastqdump
	2	metadata
	2	srafetch
	9

[Thu Nov  5 15:38:19 2020]
rule metadata:
    output: results/Metadata/SRR12845350.json
    log: info/logs/SRR12845350.metadata.log
    jobid: 3
    benchmark: info/benchmark/SRR12845350.metadata.bench
    wildcards: sample=SRR12845350

[Thu Nov  5 15:38:19 2020]
rule metadata:
    output: results/Metadata/SRR8094770.json
    log: info/logs/SRR8094770.metadata.log
    jobid: 5
    benchmark: info/benchmark/SRR8094770.metadata.bench
    wildcards: sample=SRR8094770

[Thu Nov  5 15:38:25 2020]
Error in rule metadata:
    jobid: 5
    output: results/Metadata/SRR8094770.json
    log: info/logs/SRR8094770.metadata.log (check log file(s) for error message)

RuleException:
CalledProcessError in line 71 of /home/kasper/Git_repositories/LiverDB/workflow/Snakefile.py:
Command 'set -euo pipefail;  /home/kasper/miniconda3/bin/python3.7 /home/kasper/Git_repositories/LiverDB/.snakemake/scripts/tmp2r1dpjq6.sradata.py' returned non-zero exit status 1.
  File "/home/kasper/Git_repositories/LiverDB/workflow/Snakefile.py", line 71, in __rule_metadata
  File "/home/kasper/miniconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Removing output files of failed job metadata since they might be corrupted:
results/Metadata/SRR8094770.json
[Thu Nov  5 15:38:25 2020]
Error in rule metadata:
    jobid: 3
    output: results/Metadata/SRR12845350.json
    log: info/logs/SRR12845350.metadata.log (check log file(s) for error message)

RuleException:
CalledProcessError in line 71 of /home/kasper/Git_repositories/LiverDB/workflow/Snakefile.py:
Command 'set -euo pipefail;  /home/kasper/miniconda3/bin/python3.7 /home/kasper/Git_repositories/LiverDB/.snakemake/scripts/tmpy1vn24kq.sradata.py' returned non-zero exit status 1.
  File "/home/kasper/Git_repositories/LiverDB/workflow/Snakefile.py", line 71, in __rule_metadata
  File "/home/kasper/miniconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Removing output files of failed job metadata since they might be corrupted:
results/Metadata/SRR12845350.json
Shutting down, this might take some time.
Exiting because a job execution failed. Look above for error message
Complete log: /home/kasper/Git_repositories/LiverDB/workflow/.snakemake/log/2020-11-05T153819.833894.snakemake.log
